watchfrr/watchfrr.c

```diff
@@ -1277,16 +1274,6 @@ static void netns_setup(const char *nsname)
 }
 #endif
 
-static void watchfrr_start_config(void)
```

Comment:
why the reversal of this commit?

Reply:
It's not working as expected, or I don't understand how it is supposed to work :/

Reply:
what's going wrong?

Reply:
Callbacks are not working here, should they work?

Reply:
ah did some quick investigation and the problem is that watchfrr is not getting the notifications.  
[diff.txt](https://github.com/FRRouting/frr/files/10058517/diff.txt)

Try this I bet it starts working and let's not back it out